### ISSUE TITLE
Fuse-Box path aliases not working properly

### ISSUE THREAD

**Issue by mika-fischer** [nexe/nexe#403]
Fuse-Box supports `~` as an alias for `homeDir`. For our project `homeDir` would be `src`.

However, nexe sets `homeDir` to the project root and this cannot be changed sensibly (setting cwd to src causes different failures that I don't understand).

This leads to the situation that all `require`s for modules starting with `~` are just silently ignored (which is really bad, at least I would have expected an error somewhere along the way, at the latest at runtime...).

Example:
`src/app.js`:
```
console.log('Before require');
require('~/foo');
console.log('After require');
```

`src/foo.js`:
```
console.log('foo');
```

```
$ nexe src/app.js && app
i nexe 2.0.0-rc.13
√ Compiling result
√ Entry: 'src\app.js' written to: app.exe
√ Finished in 0.554s
Before require
After require
```

**Comment by calebboyd** [comment]
Eek, Yeah. The `cwd` flag was half baked... 😶 especially with the bundler.
Its meant to run nexe in a different directory with just the cli... Which it kinda sounds like what you want.

I've fixed it up in `nexe@next`
It should work there with `nexe app.js -o ../app --cwd src`

note: even the output is now operating in `src`.

Here is where I defined the behavior if you're interested
https://github.com/nexe/nexe/blob/4ec8a43d60dec8112c97f9d504c5af8eef73f5e3/test/options.spec.ts#L17-L28

**Comment by calebboyd** [comment]
Regarding the error at runtime -- Might be an issue for fuse-box.. That is pretty strange.

In general, I'm trying not to expose the underlying bundler.

However, there are hooks eg `--bundle ./custom-bundler` that allow someone to implement their own bundle configuration.

**Comment by calebboyd** [comment]
@mika-fischer Did this work for you?

**Comment by mika-fischer** [comment]
I don't think cwd is what I want. I need more control over the bundling process.

I see that I can have that control via `--no-bundle` or `--bundle script.js`. However, with that I run into #408 ...

Currently it's too much work to get a complete bundle with all dependencies and native modules to work correctly, so I'm kind of stuck...

**Comment by calebboyd** [comment]
The home dir now correctly corresponds to the --cwd option. 

Closing this.

I believe the issues you're still experiencing can be solved by finishing #401 and #408
